src/docs_website/src/file_checker.zig

```diff
@@ -4,7 +4,7 @@ const std = @import("std");
 const log = std.log.scoped(.validate);
 const assert = std.debug.assert;
 
-pub const file_size_max = 900 * 1024;
+pub const file_size_max = 950 * 1024;
```

Comment:
That's the index, right? Maybe we should use a smaller limit in general and special-case the few large files?